abs.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/ffmpeg-feedstock/pr5/940a3b1
```

Comment:
ffmpeg 6.1.1 has been uploaded now so you can remove this. also the aggregate_check: false, as this was just because I did a pr against your branch. overlinking checks should be enabled. so the whole abs.yaml can be deleted.